frontend/src/stateless/index.ts

```diff
@@ -0,0 +1,372 @@
+import * as jsyaml from 'js-yaml';
+import _ from 'lodash';
+import { request } from '../lib/k8s/apiProxy';
+import { Cluster } from '../lib/k8s/cluster';
+import { ConfigState, setStatelessConfig } from '../redux/configSlice';
+import store from '../redux/stores/store';
+
+let indexedDBtest: any;
+
+// Import 'fake-indexeddb' only in the testing environment
+if (process.env.NODE_ENV === 'test') {
+  import('fake-indexeddb')
```

Comment:
I did not try it, let me check. Don't you think it would require much of a change in this if we change it now? It could be added in the scope of a different PR?